app/views/ViewDates.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import java.time.ZoneId
+import java.time.format.DateTimeFormatter
+
+object ViewDates {
+  def timezone = ZoneId.of("Europe/London")
+
+  val movementFormatter: DateTimeFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy").withZone(timezone)
```

Comment:
So far, dateTime formattingwas handled by injected DateTimeFormatter.
I'd advise to use same approach here. Bear in mind however, that there is one `DateTimeModule` already

Reply:
@SteveSugden Can you look at this comment? I don't know why we have this

Reply:
Yes.  It took me ages to work out where the DateTimeFormatter was magically coming from in other forms.   Surely it is better for the view to specify what formater it wants to use.  This helper class is just a place to define a format so that it can be re-used by other views if required.

Reply:
On the other hand - Dependency Injection is usual way to obtain those type of objects.

Reply:
Isn't DI more useful for injecting complex components that would make the consumer component harder to test otherwise?

I think this is just adding complexity for no apparent gain.  However...

Happy to change it if someone can explain how we over-ride the formatter returned by DateTimeModule.    Do we have an example of this.